tests/modules/programs/borgmatic/exclude-hm-symlinks-nothing-else.nix

```diff
@@ -0,0 +1,43 @@
+{ config, pkgs, ... }:
+
+let
+  backups = config.programs.borgmatic.backups;
+  excludeFile = pkgs.writeText "excludeFile.txt" "/foo/bar";
+in {
+  config = {
+    programs.borgmatic = {
+      enable = true;
+      backups = {
+        main = {
+          location = {
+            sourceDirectories = [ "/my-stuff-to-backup" ];
+            repositories = [ "/mnt/disk1" ];
+            excludeHomeManagerSymlinks = true;
+          };
+        };
+      };
+    };
+
+    test.stubs.borgmatic = { };
+
+    nmt.script = ''
+      config_file=$TESTED/home-files/.config/borgmatic.d/main.yaml
+      assertFileExists $config_file
+
+      declare -A expectations
+
+      expectations[location.exclude_from[0]]="/nix/store/18sk4zmxs1h73nbc0r2ar9z60c65mwfm-borgmatic-home-manager-exclude-symlinks.txt"
```

Comment:
Will need to read this path dynamically to avoid issues when the hash changes.

Reply:
I tried a different solution. Is it better?